services/scan/src/central_scanner_app.test.ts

```diff
@@ -329,8 +330,13 @@ test('PUT /config/package', async () => {
       'ballot-package.zip'
     )
     .expect(200, { status: 'ok' });
+
+  const ballotPackage = await readBallotPackageFromBuffer(
+    electionFamousNames2021Fixtures.ballotPackage.asBuffer()
+  );
+
   expect(importer.configure).toBeCalledWith(
-    electionFamousNames2021Fixtures.electionDefinition
+    expect.objectContaining(ballotPackage.electionDefinition)
```

Comment:
### Election Hash Testing Strategy 8
Compare against the election definition in the fixture ballot package, not the separate election definition, as the two have now diverged. 

@jonahkagan I'm wondering why the election parser isn't failing though? Maybe it doesn't care about extraneous fields. I can replace this fixture if you recommend, but I'd like to avoid replacing it for all upcoming changes.

Reply:
Hmm, I'm not totally sure I grok this at first glance. Where would you expect a failure?

Reply:
The election definition within the ballot package still has `contest.section` in it. So one might think that on parsing that election definition, there would be a `Zod` error because it has fields not in the schema. But I've never been sure if `Zod` cares about extraneous fields.

Reply:
> By default Zod object schemas strip out unrecognized keys during parsing. You can disallow unknown keys with .strict() . If there are any unknown keys in the input, Zod will throw an error.

https://github.com/colinhacks/zod#strict